specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v2.0/LUIS-Runtime.json

```diff
@@ -5,11 +5,52 @@
     "version": "2.0"
   },
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/luis/v2.0",
-    "useSchemePrefix": false,
+    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.{AzureCloud}/luis/v2.0",
     "parameters": [
       {
-        "$ref": "#/parameters/Endpoint"
+        "name": "AzureRegion",
+        "description": "Supported Azure regions for Cognitive Services endpoints",
+        "x-ms-parameter-location": "client",
```

Comment:
Is there a reason these are supposed to be client level parameters?

Reply:
@nebadr @AmirGeorge - client level params?

Reply:
@tsuwandy for possible input.

Reply:
@dsgouda We are not aware exactly why but is it causing problem?

Reply:
This is not necessarily a problem. We want to ensure your intent is to tie each client instance to a particular Azure instance.
Please confirm the same.

Reply:
@dsgouda Yes, this is our intent. Confirmed.